### PR TITLE
study2

### PR DESCRIPTION
**Pull request by huangyulong1988**
INTRODUCTION 
Idesia polycarpa Maxim. is a deciduous tree of the Salicaceae family,1 which mainly originates from Asian countries such as China, Korea and Japan.2 The oil from its fruit has been used as cooking oil for many years in China due to its health benefits and therapeutic effects.3 Similarly, the seeds of this tree have been used as an insecticide in agriculture in Korea.2 Meanwhile, the oil residue of its fruits can also be used as a high quality raw material of livestock feed.	 Previous studies have reported that many hydroxycinnamic acids derivatives were isolated from the fruits of the plant such as idescarpin, idesin, idesolide, phenazine derivatives, and salirepin,2, 4, 5 which showed anti-oxidative, skin-whitening and anti-inflammatory effects, anti-aging, and anti-adipogenic activity.3, 6-8 In addition, its leaves received particular attention as herbs and foods because of the remarkable bioactive, especially the hemostatic activity.2 Given that the leaves can benefit human health, it was used as a type of tea for many years in some region of China such as In some areas of Sichuan and Chongqing provinces. More recently, some research reported that the leaves of the plant also contain considerable levels of bioactive compounds which were hydroxycinnamic acid derivatives.1 
Hydroxycinnamic acids are widely distributed in plants as secondary metabolites and one of the major classes phenolic compounds.9 The most common hydroxycinnamic acids are sinapic acid, caffeic acid, coumaric acid, and ferulic acid. However, the hydroxycinnamic acids as a moiety of its derivatives rarely exist in free form and commonly occur as O-glucosides, or as O-acylglucoses and O-acylquinic acids in plants.9, 10 It was reported previously that the hydroxycinnamic acids and its derivatives are attributed with wide spectrum of biological activities in vitro and in vivo, including anti-oxidation, anti-bacterial, and anti-mutagenic properties.11-13 More recently, published articles on isofraxidin, a coumarin component reduced lipid accumulation in human hepatoma cells (HepG2).14 Our previous study found that hydroxycinnamic acid derivatives from the leaves of I. polycarpa Maxim., which exhibited significant inhibition of the MCF, and A549 cells proliferation (data not shown). Therefore, hydroxycinnamic acids and its derivatives have deserved more attention in the both field of foods and herbs.15, 16 In addition, hydroxycinnamic acids with its derivatives have both E (trans-form) and Z (cis-form) form exist in dietary plant derived foods and herbs, but the E form is predominant in natural and also deemed highly unstable.17 Many studies have reported that heat and light may induce degradation or change of the hydroxycinnamic acids and its derivatives,10, 13, 17 whereas the biological activity of phytochemicals might be affected due to the alteration of stereo form or functional group in chemical structure.18 For the reason, it is an enormous challenge for processing of the herbs and foods products, particularly to prevent the active ingredients loss in the drying processing.
Drying has been widely used in the field of foods and herbs for the initial processing as a traditional method,19, 20 which not only decrease degradation and prolong the shelf-life of herbs and foods but also prevent considerable loss of bioactive compounds during removing the moisture.21, 22 However, as a result of the different drying method and treatment conditions, drying may bring about damage to the bioactive compounds of herbs and foods. Therefore, it is a critical step to choose a drying method and optimization of drying process for the bioactive compounds retention.23 There are a few drying methods commonly used to processing the herbs and foods such as sun-drying, shade-drying, oven-drying, and freeze-drying. Due to the low production cost and flexibility of sun-drying, it is the most widely used method to extend plant products shelf life by reduced the content of moisture,24, 25 while the low drying rate prolongs drying time may cause the significant quality degradation and change in the photoactive compounds of sun-drying products.21 And the shade-drying is also a low production cost but which mainly applied to processing the special materials that contain abundant photoactive compounds as a result of its low drying rate.19 With respect to the oven-drying, it is frequently used for processing various products such as herbs and plant foods because of its efficient and easy for industrialized production, but the high temperature may lead to degradation and change in thermally labile compounds.21 Freeze-drying is supposed to be the best drying method compare to other methods because it can preserve various labile compounds of herbs and foods,20, 25 while this method is very expensive because of it need to maintain a low temperature and vacuum condition during the processing.26, 27 Although many studies reported that the effect of different drying method and treatment conditions on the total phenolic content of plant material,16, 20, 21, 28 there were few reports about the change in the structure-activity relationship of individual compounds of total phenolic after treatment by those methods.
The objective of this study is the first to investigate the effects of various drying methods on individual compounds of hydroxycinnamic acid derivatives in leaves by freeze-drying, shade-drying, oven-drying, and sun-drying. Furthermore, as herbs in its initial stage of developmental process, it is important to shed light on the physiochemical properties as well as the relationship between structure and activity of individual compounds of leaves, including the factors and mechanisms which influence its activity and stability. Consequently, according to above study, chose a suitable drying method for the drying processing of I. polycarpa Maxim. leaves.
MATERIALS AND METHODS
Material. All solvents used in the present study were of analytical grade and were purchased from Chengdu Kelong Reagent Co., Ltd. (Chengdu, China). Formic acid (HPLC grade) was obtained from Chengdu Kelong Reagent Co., Ltd. (Chengdu, China). Acetonitrile and methanol (HPLC grade) were from Swell Scientific Instruments Co., Ltd. (Chengdu, China). Ultrapure water (18.25 MΩ) was prepared using an ultrapure-water system (WSD, Chengdu, China). Oleic acid (OA), Bovine serum albumin (BSA), oil red O, and 3-(4, 5-Dimethylthiazol-2-yl)-2, 5- diphenyltetrazolium bromide (MTT) were purchased from Sigma-Aldrich (Shanghai, China). Idesia polycarpa Maxim. leaves were collected in Jun 2018 in campus of Sichuan University (Sichuan, China) and authenticated by A.P. Jie Bai, School of Life Sciences, Sichuan University. The voucher specimen (No. 0071614) was deposited in the Herbarium of Sichuan University.
Drying Methods. Before extraction, took the same amount of leaves (100g) and dried with different methods. For the sun drying process, the fresh leaves were distributed on a glass plate and exposed to sunlight at a temperature between 40 C–50 C and a relative moisture of 49%–55% (Temperature/Humidity datalogger, RHT10, Shanghai, China) for about 20 h of daylight under Chengdu environmental conditions during Jun 2018 (Chengdu, China). In the shade drying process, the fresh leaves were dried in a shady and drafty room (20 C, relative humidity of  49%–55%) for 48 h. Oven drying was dried in an oven-drier (Shanghai Pudong Rong-feng Scientific Instrument Co., Ltd., Shanghai, China) at 60 °C for 8 h, and with a constant air-flow rate of 2.0 m/s. For the freeze-drying process, the leaves were frozen in a metal plate at -20 C for 2h, and then dried using a freeze dryer (Ningbo Shuangjia Yi Qi Co., Ltd., Ningbo, China) equipped with a vacuum chamber under a reduced pressure of 0.1 Pa at -80 °C for 24 h. After the drying processing, the moisture contents of dried samples was measured by oven drying at 105 °C.29 And then those dried samples were collected into sealed plastic bags, and stored in the dark at -20 °C until analysis.
Extraction of Leaf Samples. After processing by four drying methods, all samples were grinded into fine powder. 0.1 mg of the Idesia polycarpa Maxim. leaves were extracted using 4 mL of aqueous ethanol solution (30% v/v) with assisting an ultrasound at 59 Hz and 20 C for 30 min, and each sample was repeated three times. The crude extract solution was centrifuged (12000 rpm) at 4 C for 5 minutes, and then filtered through a 0.22 μm membrane and analyzed by UPLC. Subsequently, the crude extract of 100 mg of the Idesia polycarpa Maxim. leaves were concentrated under reduced pressure by rotary evaporation at 45 C, and then the extract was re-suspended in redistilled water (200 mL) and partitioned with ethyl acetate (10 times) by liquid–liquid partitioning. All of the ethyl acetate extracts were concentrated under reduced pressure by rotary evaporation at 45 C to obtain the partially purified sample, the ethyl acetate fraction, which was then stored at 4 C for further experiments. And then those compounds 1, 2, 3, and 4 of ethyl acetate fraction were isolated and purified from the ethyl acetate fraction by HSCCC and prep-HPLC, and which purities over 98%.
The Stability Analysis of hydroxycinnamic acid derivatives. Compounds 1, 2, 3, and 4 are the main active ingredient of Idesia polycarpa Maxim. leaves. In order to investigate the stability of the four compounds, those compounds were treated with different waves of UV and temperature. The four compounds (0.5 mg/mL) were exposed to different waves of UV lights (256, 302, and 365 nm) for 0−60 min to examine the light stability. And the four compounds (0.5 mg/mL) were treated with different temperature (-20, 0, 20, 40, 60, 80, and 100 C) for 0.5−4 h to examine the heat stability. After processing, those samples were filtered through a 0.22 μm membrane and analyzed by UPLC. And each experiment was repeated three times.
UPLC−PDA−ESI−MS Analysis. All of the hydroxycinnamic acid derivatives were analyzed with UPLC-QDa (Waters Corp., Milford, MA, USA). The apparatus was a Waters Acquity UPLC H-Class system equipped with an auto-sampler, a quaternary solvent manager system, and a photodiode array detector (PDA) detector and an Acquity QDa single quadrupole mass detector equipped with an ion-spray interface. 
The UPLC analysis was performed using an Acquity UPLC HSS T3 column (100 mm × 2.1 mm I.D., 1.7 μm, waters), and the temperature of autosampler and column were set at 25 °C and 40 °C. The mobile phases were (a) acetonitrile and (b) 0.1% formic acid and 25 mM ammonium formate aqueous solution. The gradient was as follows: 10% (a) for 0.5 min, 10%–25% (a) in 1 min, 25%–30% (a) in 1.75 min, 30%–45% (a) in 3.5 min, and 45% (a) for 0.25 min, followed by re-equilibration of the column for 2.5 min with 10% (a). The flow rate was 0.5 mL/min and the injection volume was 0.5 μL. The detection wavelength was set at 312 nm. The dried crude sample (0.5 mg) was dissolved in 1 mL of acetonitrile and then filtered through a 0.22 μm membrane and analyzed by UPLC. Similarly, those compounds were prepared for UPLC analysis in the same manner as the crude sample. 
After the PDA, a flow of 0.5 mL/min was directed to the mass spectrometer. For LC-MS analyses, the ESI interface was operated in the positive and negative ion mode, with the following settings: capillary voltage was ± 0.8 kV, sampling rate was 8 Hz, probe temperature was 600 C and cone voltage was set at 15 V, respectively. Both nebulizer gas (gas 1) and heater gas (gas 2) were set at 50 psi while the curtain gas was kept at 30 psi. Nitrogen was used as a nebulizer and auxiliary gas. An m/z scan range was collected from 50 and 600.
Cell Culture. The human liver-derived cell line HepG2 was obtained from Chinese Academy of Sciences (Shanghai, China). The cells were maintained in Hyclone Dulbecco’s Modified Eagle’s Medium (DMEM; Thermo Scientific, Rockford, IL, USA) supplemented with 10% ultra-low endotoxin Fetal Bovine Serum (Cell-Box, South America Origin), 100 IU/mL penicillin, and 100 IU/mL streptomycin. Cells were incubated at 37 °C in a 5% CO2 humidified atmosphere, until the culture reached 90% confluency.30 
Cell Viability. Cell viability was measured using the MTT assay.14 The HepG2 cells were seeded in 96-well plates (1 × 104 cells per well) for 12 h at 37 °C and treated with various concentration of olecid acid (0.25, 0.5, 1 and 2 μΜ) for 24h, and then compounds 3, 4 and 6 (1.25, 2.5, 5, 10, 20, 40, 80 μΜ) were added into cell culture medium for 24h. Then cell viability was determined by colorimetry with 100 μL of DMEM containing 5 mg/ml MTT at 37 C for 4 hours. After 150 μL DMSO was added for 10 minutes, the optical density was read at 490 nm using a microplate spectrophotometer system (Spectra Max M2, Molecular Device, Sunnyvale, CA, USA); complete medium used was as a blank control. Each assay was replicated in five wells, and each experiment was repeated three times. 
Oil Red O Staining and Lipid Droplets Analysis by Cell Imaging. HepG2 cells in the logarithmic growth phase were seeded in 6-well plates at a density of 2 × 105 cells/mL. After 12 h, the cells in 6 well culture plate were washed in PBS and added with medium containing 0.5 mM oleic acid-bovine serum albumin (OA-BSA) complex in the presence or absence of compounds 1, 2, 3, and 4 (20μΜ) for 24h. The medium with only BSA was selected as the control. 
To measure total intracellular lipid content, HepG2 cells were stained by the Oil Red O (ORO) method.30 Briefly, after treatments, cells were washed three times with PBS and fixed with 4% paraformaldehyde for 20 min at room temperature (RT). Subsequently, the cells were stained for 40 min in a freshly filtered 0.5% ORO solution (Sigma-Aldrich, Shanghai, China) at RT. In order to remove the background staining, the cells were washed with 60% isopropanol solution for 20 s. Lipid droplet accumulation was detected by watching under the microscope, the image of each group was photographed. To quantify intracellular lipid accumulation, isopropanol was added to each sample shaken at room temperature for 10 min. The extracted dye was removed by gentle pipetting and its absorbance was monitored by microplate reader at 510 nm. 
Statistical Analysis. The IBM SPSS Statistic program (version 22) was used for the all statistical analyses. In this article, all data were performed in triplicate using three independent experiments. This data of the four compounds in different condition (different waves of UV lights and temperature) at all the time points were analyzed using one-way ANOVA, and which was used also to analysis the changes in the content of various groups of compounds and lipid droplets of cells. P < 0.05 was considered that the data had significant differences by Duncan's multiple range test. And the graphics were generated using the GraphPad Prism (version 5.0).
RESULT AND DISCUSSION
The effect of drying methods on the compounds of I. polycarpa Maxim. leaves. In this study, the I. polycarpa Maxim. leaves were processing by freeze-drying, shade-drying, oven-drying, and sun-drying, and then the ethanol extracts of the four samples were analyzed by UPLC. The result demonstrated that those four drying methods have a significant effect on the area of four UPLC peaks (1, 2, 3, and 4) (Figure 1 A−D). According to many previous studies,21, 26 freeze-drying is regarded as the best drying method for retaining active compounds than other method, which can prevent this degradation and oxidation of unstable compounds due to the low temperature and vacuum condition. The content of compounds 3 and 4 were declined but not significant under the shade-drying processing and then decreased by 19.72% and 18.69%, respectively, after treatment with oven-drying (Figure 2). In contrast to compounds 3 and 4, however, the content of compound 1 was increased by 1-fold and 4.1-fold, respectively, after drying by shade-drying and oven-drying, and compound 2 increased by 15.78% and 104.26%, respectively. Above result indicate that compounds 3 and 4 appear to be more heat labile than compounds 1 and 2. Also, after sun-drying processing, the content of compounds 1, 2, and 4 increased by 57.60%, 58.42%, and 25.33%, respectively, and compound 3 showed a 22.47 % loss compared to freeze-drying processing, indicating compound 3 more susceptible to losses during the sun-drying processing, while the sun-drying processing may also promote other compounds transform into compounds 1, 2, and 4. These results are consistent with previous studies reporting more marked change in the phenolic compounds of plant material during the shade-drying, oven-drying, and sun-drying processing than freeze-drying processing because of the high temperature might result in enzymatic and nonenzymatic degradation of compounds in plant materials.15, 24, 31, 32 Noticeably, the content of total phenolic and flavonoids of plant samples as the main indices to choose the drying method in many previous researches,16, 32, 33 and there are few studies have been exhibited the mechanism of monomeric compounds change in plant material during various drying methods. 
Identification of Unstable Components. Previous study has identified 15 compounds of the I. polycarpa Maxim. leaves, and there are four pair of E/Z isomers of hydroxycinnamic acid derivatives among them.1 Thus, it is difficult to identify accurately each compound of the crude extracts of I. polycarpa Maxim. leaves only by LC-MS as result of those isomers have the same molecular mass. To investigate the effects of various drying methods on the mechanism of monomeric compounds change in leaves, the four compounds must to be identified. In the experiment, the crude extracts were separated into two fractions via the initial separation of ethyl acetate due to the large amount of compounds of leaves. Interestingly, the major compounds of the ethyl acetate fraction of leaves were those four compounds after analyzed by LC-MS (Figure 3 A) which also exhibited that those compounds have the same molecular mass with m/z 417 under the negative ion mode (Figure 3 B). Thus, we inferred the four compounds may be the isomers of each other. Previous studies reported that the extraction and isolation of isomers from natural sources was necessary and highly valuable to research in natural medicinal chemistry.13, 34-37 In order to further analyze their physical and chemical properties, the four compounds were isolated and purified by high-speed counter-current chromatography (HSCCC) combined preparative high-performance liquid chromatography (prep-HPLC) (data not shown), and their purities were over 98% (Figure 4). And then the data of FI-IR, MS, and NMR techniques confirmed the structure of those four compounds (1, 2, 3, and 4) as 1-[(6ʹ-O-(E)-p-coumaroyl)-β-D- glucopyranosyl]-oxy-2-phenol (1),38 1-[(6ʹ-O-(Z)-p-coumaroyl)-β-D-glucopyranosyl]-oxy-2- phenol (2), 1-[(4ʹ-O-(E)-p-coumaroyl)-β-D-glucopyranosyl]-oxy-2-phenol (3),1, 38 and 1-[(4ʹ-O- (Z)-p-coumaroyl)-β-D-glucopyranosyl]-oxy-2-phenol (4) (Figure 8),1 respectively, namely four isomers of hydroxycinnamic acid derivatives. 
The Stability Analysis of Four Hydroxycinnamic Acid Derivatives. The processing conditions play an important role in the plant materials, such as light and temperature which are the mainly factors that induce degradation and change in the structural of bioactive compounds.19, 39 However, many previous study tend to focuses on the processing methods which should minimizing large the total phenolic content of plant materials.24-28, 40 For this very reason, little effort has been devoted to study the change mechanism of monomeric compounds in plant materials. In order to investigate the effect of light and temperature on the four isomers, the methanol solution of the four isomers were treated with UV radiation and temperature for various times, and then analysis by UPLC.
In this present study, the light-stability of E-isomers of compounds 1 and 3 were investigated, which irradiated by UV (254, 302, and 365mnm) for various time. As previous study,13, 36 Z-form of compounds 1 and 3 were converted into the corresponding Z-isomers of compounds 2 and 4, respectively, and the content of the two compounds decreased in a linear manner as the UV (254, 302, and 365mnm) irradiation with time increased. After irradiation of UV (254, 302, and 365mnm) for 50 minutes, the content of compound 1 was reduced to 65.67%, 54.29%, and 68.04%, respectively, (Figure 5 A) and the corresponding conversions of compound 2 were increased by 34.86%, 45.19%, and 33.05% , respectively (Figure 5 C). Similarly, the content of compound 1 was also reduced to 64.48%, 56.29%, and 67.91%, respectively (Figure 5 B), and the reduced amount of compound 3 was nearly same as the increased content of compound 4 (Figure 5 D). The result indicated that the UV has the greatest effect on the compounds of 1 and 3 at 302 nm compared to the other two wavelengths (P < 0.05). As previous study, the derivatives of (E)-p-coumaric acids were unstable under light conditions, which would be converted into the corresponding Z forms by light.10, 13, 41
With regard to heat-stability, the four isomers solution in methanol was incubated at various temperature (-20 C−100 C) for 0.5−4 h, and then analyzed by UPLC. Most interestingly, Figure 6 A and B revealed that the content of compounds 1 was not changed at 60 C for 4 h, and the compounds 2 stayed fairly stable at 80 C for 4 h until the temperature beyond 80 C. However, the Figure 6 C and D exhibited that the compounds 3 and 4 were very sensitive to heat as the temperature exceeded 20 C and the conversion of compounds 3 and 4 into compounds 1 and 2, respectively, with the temperature and time increased. And these contents of compounds 3 and 4 drastically lower with the increase in treatment temperature and time, which were minimized to 2.45% and 2.89%, respectively, at 100 C for 4 h (Figure 6 C and D). Notably, as show in the Figure 6 E and F the content of compounds 1 and 2 increased to the highest point of 53.1% and 53.34%, respectively, at 100 C for 2 h. According to the result of experimentation, heat not induced transformations from E-(1)and E-(3) to Z-(2) and Z-(4), but which could conversion of 4-O-acylglucoses of compounds 3 and 4 into the stable 6-O-acylglucoses of compounds 1 and 2. Hence, these results can also account for some of increasing the content of compounds 1 and 2 in leaves after processing by shade-drying and oven-drying. In addition, our results were not in agreement with observations of other researchers that temperature may lead to the degradation of phenolic compounds in plant products.25, 33 Overall, the conversion of these four isomers in processing of UV and heat demonstrated that heat and light may be were the major factors, which induce the conversion of compound 3 into compounds 1 and 4, respectively, and transformations from the compounds 1 and 4 to compound 2 during the sun-drying processing of leaves (Figure 7).
Activity Analysis of The Four Hydroxycinnamic Acid Derivatives. In our previous study, the ethyl acetate extracts of crude extract of leaves showed a significant activity to decrease hepatic steatosis in HepG2 cells which was induced with oleic acid (0.5 mM).30, 42 However, the ethyl acetate friction mainly contained four hydroxycinnamic acid derivatives (1, 2, 3, and 4), according to the UPLC analysis (Figure 3 A). Furthermore, hydroxycinnamic acid derivatives are known to have a wide array of bioactivity, such as anti-diabetes, which is close relationship with structure according to previous studies.13, 37 To estimate the effect of the four isomers on the cell viability of HepG2 cells, the cells were treated with different concentrations of the four compounds (1.25, 2.5, 5, 10, 20, 40, and 80 μM) for 24 h. And MTT assay indicated that no significant in cell numbers when the concentration of the compounds 1, 3, and 4 were equal to or below 40 μM (Figure S1). Interestingly, the result showed that observable loss of cell viability occurred when the concentration of compound 2 was beyond 20 μM (Figure S1). Therefore, we chose the four isomers (20 μM) combined with 0.5 mM OA to treat the HepG2 cell in the following experiment. Figure 8 displays that OA treatment for 24 h significant increased the lipid content in the HepG2 cell as determined by intracellular lipid drip assay and observed by Oil Red O staining under inverted fluorescence microscope. After these four isomers treatment for 24 h, the result exhibited that compounds 2 and 4 treatments significantly reduced the intracellular lipid level by 65% and 47% (P<0.05), respectively, and compound 3 treatments reduced by 10.69%, but compound 1 no presented a significant activity to decrease the lipid content in the HepG2 cells. These different lipid-lowering activities of four isomers indicated that the activity of Z-form compounds significant decreased the lipid accumulation in cells compare with E-form compounds consistent with previous studies that Z-form exerts a higher biological activity than E-form in many aspects.13, 43-45 In addition, we also found that the 4-O-acylglucoses moiety increased the lipid-lowering activity of E-(3) compare with 6-O-acylglucoses of E-(1), and the 6-O-acylglucoses moiety increased significantly the lipid-lowering activity of Z-(2) compare with 4-O-acylglucoses of Z-(4), indicating that the 6-O-acylglucoses moiety increased the activity of Z-form to decrease the hepatocyte lipid accumulation in OA-induced steatosis conditioned HepG2 cells, and the 6-O-acylglucoses moiety declined the lipid-lowering activity of E-form. Therefore, the order of lipid-lowering activity was as follows: Z-(2) > Z-(4) > E-(3) > E-(1). Our results are consistent with previous studies reporting isofraxidin (100μM) of hydroxycinnamic acid derivatives significant declined the lipid accumulated in HepG2 cells, indicating that compounds 2, 3,and 4 may also improves the OA induced lipid accumulation of HepG2 cells by down-regulation of inflammatory cytokine expression such as NF-κB , TNFα and IL-6.14
In summary, heat and light may were more susceptible to change in hydroxycinnamic acid derivatives of leaves. Hence, the sun-drying processing of leaves had a greater effect on the hydroxycinnamic acid derivatives by light and heat than freeze-drying, shade-drying, and oven-drying. Moreover, these compounds 1 and 3 were not degradation but conversion into corresponding isomers 2 and 4 which had a greater activity to decline the OA-induced lipid accumulation in HepG2 cell than isomers 1 and 3. Therefore, the sun-drying processing increased the content of bioactive compounds of I. polycarpa Maxim.leaves compare with the other three methods and was an excellent drying method during the processing of leaves.

### CODE REVIEW

**Comment by markfullmer** [comment]
Closing -- spam.
